white-paper.md

```diff
@@ -48,16 +48,16 @@ We propose an immutable decentralized registry designed to distribute value base
 
 Additionally, we believe that material information must be available via the package manager for developers to assess whether they can trust a package and its author. This information may be based on reputation, community kudos, data retrieved from decentralized identity (DID[^4]) systems, other package managers, or incentive mechanisms that potentially rely on network participants putting economic value at risk.
 
-We predict that tea’s combination of tools, information, and rewards will justly incentivize developers, helping stimulate the growth of open-source software and fostering innovation.
```

Comment:
"justly" was a departure from the usage of "equitable" and "fair" everywhere else.

Reply:
According to the dictionary, both are synonymous. "according to what is morally right or fair; fairly: we deal justly with complaints." - Justly should not be seen as a departure

Reply:
Yeah, that's just. I won't contest that they're not synonyms. 😁 

When all is said and done here, I'll make another pass over this pull request and adjust according to all your feedback. I super appreciate you looking at it. 🎉 


I think "just" has a strong legal connotation than "fair", but I'm also partial to the [Webster's 1913 dictionary definition](http://www.websters1913.com/words/Justly):

> In a just manner; in conformity to law, justice, or propriety; by right; honestly; fairly; accurately.